walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc

```diff
@@ -452,13 +343,64 @@ Conflicting description from laptop
 
 . Bring both of your devices back online, a popup window should appear warning you about conflicts.
 
-
 [type=verification]
 ****
 Did the tasks sync as expected?
 ****
 
 [type=verificationFail]
 ****
-{standard-fail-text}
+Verify that you followed each step in the procedure above.  If you are still having issues, contact your administrator.
 ****
+
+. Close terminal window running server application
+
+[time=15]
+== Add authentication and authorization to the Data Sync application using Red Hat SSO
+
+In this task, we will configure both the frontend and the backend of the
+{data-sync-starter} with the {customer-sso-name}.
+
+DataSync starter has authentication and autorization enabled out of the box.
+Developers need to configure server and client application to use their keycloak instance
+and add required authorization rules to their model.
+
+=== Add authorization rule for Task deletion
```

Comment:
This is outdated. 

We can just say:

Please review auth file (https://github.com/aerogear/datasync-starter/blob/master/server/src/config/auth.ts#L8). We can see that only admin can delete task. 
For more information check https://graphback.dev/docs/authentication/keycloak